src/components/ui/Card.tsx

```diff
@@ -99,7 +102,13 @@ const cardContainer = ({
       'py-8': typeof paddingType === 'undefined',
       'py-small': paddingType === 'small',
     }),
-    useAppStyles ? appStyles : thin ? thinWidthStyles : defaultWidthStyles,
+    useAppStyles
+      ? appStyles
+      : thin
+      ? thinWidthStyles
+      : wide
+      ? wideWidthStyles
+      : defaultWidthStyles,
```

Comment:
Can we split it somehow? It's harsh to understand what's happening here in this `monster` 😅